### PR TITLE
Update AutoML docs to show installation from dashboard

### PR DESCRIPTION
**Pull request by unmonoqueteclea**
Since yesterday, BigML Dashboard allows the creation (and import) of WhizzML scripts with optional resource input parameters (thanks @joseribes! ), so now AutoML can be fully installed from the dashboard. It was something requested by some users. 

As WhizzML libraries are not used by many scripts, I have added detailed instructions about their installation

### CODE REVIEW

**Comment by unmonoqueteclea** [comment]
@mmerce I added a new feature that you requested me some time ago. If the user doesn't provide a validation-dataset, AutoML will create one from the training dataset. This is configurable with a new parameter in the configuration.

**Comment by unmonoqueteclea** [comment]
Last thing in this PR, I promise, I added a new configuration parameter `balance-objective` for ensembles created during feature selection step

**Comment by unmonoqueteclea** [comment]
@mmerce ping

File: automl/readme.md [hunk 1]
```diff
@@ -17,24 +17,63 @@ script.
...
+Script`](./automl-script) from Github as you would do with other
+WhizzML scripts.
+
+![Import Script](./res/import-script.png)
+
+You may see the following error
+
+![Import Library Error](./res/lib-error.png)
+
+In that case, you will have to find, in the selector, the library you
```
Comment (mmerce) [inline automl/readme.md]: In that case, import the library that you created in the previous step by selecting it with the drop down selector.

File: automl/readme.md [hunk 2]
```diff
@@ -17,24 +17,63 @@ script.
...
+uploaded previously. After selecting it the error will disappear.
+
+After these steps, you will be able to import the script correctly into your
```
Comment (mmerce) [inline automl/readme.md]: Following these steps, you should be able to create the script in your BigML account.

File: automl/readme.md [hunk 3]
```diff
@@ -17,24 +17,63 @@ script.
...
+[bigmler](https://bigmler.readthedocs.io/en/latest/) installed in your
+system, just check out the [WhizzML examples
 repository](https://github.com/whizzml/examples/) and, at its [top
 level directory](https://github.com/whizzml/examples/), issue the
 command:
 
-    make compile PKG=automl
+     bigmler whizzml --package-dir=automl
+
+That will create all necessary library and script resources for you.
```
Comment (mmerce) [inline automl/readme.md]: That will create both the AutoML library and script resources for you.

**Review by mmerce — CHANGES_REQUESTED** [review]

**Comment by unmonoqueteclea** [comment]
Ready...

**Review by mmerce — APPROVED** [review]

**Pull request merged**